aspnetcore/mvc/controllers/filters.md

```diff
@@ -178,97 +222,123 @@ In the following code, both the `ShortCircuitingResourceFilter` and the `AddHead
+VS debug window shows 
+FiltersSample.Filters.LogConstantFilter:Information: Method 'Hi' called
+-->
 
 ## Authorization filters
 
 *Authorization filters*:
 
+* Are the first filters run in the filter pipeline.
 * Control access to action methods.
-* Are the first filters to be executed within the filter pipeline. 
-* Have a before method, but no after method. 
+* Have a before method, but no after method.
+
+Custom authorization filters require a custom authorization framework. Prefer configuring the authorization policies or writing a custom authorization policy over writing a custom filter. The built-in authorization filter:
 
-You should only write a custom authorization filter if you are writing your own authorization framework. Prefer configuring your authorization policies or writing a custom authorization policy over writing a custom filter. The built-in filter implementation is just responsible for calling the authorization system.
+* Calls the authorization system.
+* Does not authorize requests.
 
-You shouldn't throw exceptions within authorization filters, since nothing will handle the exception (exception filters won't handle them). Consider issuing a challenge when an exception occurs.
+Do **not** throw exceptions within authorization filters:
+
+* The exception will not be handled.
+* Exception filters will not handle the exception.
+
+Consider issuing a challenge when an exception occurs in an authorization filter.
 
 Learn more about [Authorization](xref:security/authorization/introduction).
 
 ## Resource filters
 
-* Implement either the `IResourceFilter` or `IAsyncResourceFilter` interface,
-* Their execution wraps most of the filter pipeline. 
-* Only [Authorization filters](#authorization-filters) run before Resource filters.
+* Implement either the <xref:Microsoft.AspNetCore.Mvc.Filters.IResourceFilter> or <xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncResourceFilter> interface.
+* Execution wraps most of the filter pipeline.
+* Only [Authorization filters](#authorization-filters) run before resource filters.
 
-Resource filters are useful to short-circuit most of the work a request is doing. For example, a caching filter can avoid the rest of the pipeline if the response is in the cache.
+Resource filters are useful to short-circuit most of the pipeline. For example, a caching filter can avoid the rest of the pipeline on a cache hit.
 
-The [short circuiting resource filter](#short-circuiting-resource-filter) shown earlier is one example of a resource filter. Another example is [DisableFormValueModelBindingAttribute](https://github.com/aspnet/Entropy/blob/rel/1.1.1/samples/Mvc.FileUpload/Filters/DisableFormValueModelBindingAttribute.cs):
+Resource filter examples:
 
-* It prevents model binding from accessing the form data. 
-* It's useful for large file uploads and want to prevent the form from being read into memory.
+* [The short circuiting resource filter](#short-circuiting-resource-filter) shown previously.
```

Comment:
It's the first time I've noticed, but usually short circuit is written as short-circuit. There are a few other cases where the - is missed off but generally it seems to be there.